docs-source/content/userguide/managing-domains/configoverrides/_index.md

```diff
@@ -366,6 +371,14 @@ Incorrectly formatted override files may be accepted without warnings or errors
       * This directory, or a subdirectory within this directory, should contain each of your custom situational configuration files.
       * If it doesn't, then this likely indicates your domain resource `configOverrides` was not set to match your custom override configuration map name, or that your custom override configuration map does not contain your override files.
 
+* If the Administration Server pod do start but fails to reach ready state:
+  * Check if there is a message ` WebLogic server failed to start due to missing or invalid situational configuration files` in the Administration Server pod's `kubectl log`
+    * This suggests that the Administration Server failed to start may be caused by errors found in the a configuration override file.
+      * Lines containing the String `<Warning>` may be found in the Administration Server pod log to provide more hints. 
```

Comment:
I think it's better to search for `situational` instead of `<Warning>`, or at least in addition to Warning, as this will catch all types of log messages (Error, Critical, etc.)    (Also because I've noticed that Warnings are extremely common in some customers and layered products, even though the system is running acceptably as far as the admin is concern, so reading through all Warnings can be a bewildering exercise trying to figure out which ones are important.)

Reply:
Agree